spec/swagger.yaml

```diff
@@ -96,6 +96,16 @@ tags:
   - name: "Context"
     description: |-
       Visual context helps Translators make linguistic, layout, and spacing decisions based on where strings appear in your mobile or desktop application. The Context API supports uploading various forms of context (images and HTML), and associate the context with content in Smartling.
+      
+      **Context API Usage Limitations**:
+      * Maximum request size is 25MB.
+      * Maximum file upload size is 20MB.
+      * Maximum HTTP header size is 32KB.
+      * For bindings requests that use pagination, maximum page size is 5000 elements.
+      * For contexts requests that use pagination, maximum page size is 100 elements.
+      * Rate limits:
```

Comment:
I would not specify API limits here like concurrency and max rate per sec (i think it should be removed from the others we have documented). The reason is we have a page that talks about receiving back a 429 and that they should wait and try again w/ an exponential retry. And I think it's better if clients use that approach rather then them trying to code under an API limit we might want to lower. That's my 2 cents.

Reply:
UPDATED